src/app/shared/entry.ts

```diff
@@ -461,6 +461,18 @@ export abstract class Entry implements OnDestroy {
    * @param topicId The ID of the topic on discourse
    */
   discourseHelper(topicId: number): void {
+    // The #discourse-comments element, included in the entry component's html template,
+    // marks the location where the discourse embed script will add the discourse link.
+    // If the #discourse-comments element does not exist, or if the previous topicId
+    // (from the previous call) is the same as the requested topicId, no dothing.
```

Comment:
Typo: `no dothing` :slightly_smiling_face:

Reply:
That's my trial balloon of "Olde English" comments, hackernews sez they're gonna be the next big thing...